pkg/apis/metal3/v1alpha1/baremetalhost_types_test.go

```diff
@@ -529,3 +529,128 @@ func TestCredentialStatusMatch(t *testing.T) {
 		})
 	}
 }
+
+func TestGetImageChecksum(t *testing.T) {
+	for _, tc := range []struct {
+		Scenario string
+		Host     BareMetalHost
+		Expected bool
+	}{
+		{
+			Scenario: "both checksum value and type specified",
+			Host: BareMetalHost{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "myhost",
+					Namespace: "myns",
+				},
+				Spec: BareMetalHostSpec{
+					Image: &Image{
+						Checksum:     "md5hash",
+						ChecksumType: "md5",
```

Comment:
Prefer to use the constants here.

Reply:
Thanks. Making the change. It does look nicer as a constant.